File: python/tree/linked_list/ListNode.py
class ListNode:
    def __init__(self, val=0, next=None) -> None:
        self.val = val
        self.next = next

    @staticmethod
    def print(head: "ListNode") -> None:
        print('[', end='')
        while head.next:
            print(head.val, end=", ")
            head = head.next
        print(head.val, end="]\n")

File: java/libraries/linked_list/ListNode.java
package libraries.linked_list;

public class ListNode {
  public int val;
  public ListNode next;

  public ListNode(int val) {
    this.val = val;
    this.next = null;
  }

  public ListNode(int val, ListNode next) {
    this.val = val;
    this.next = next;
  }

  public static void println(ListNode head) {
      if (head == null) {
          System.out.println("[]");
      }

      System.out.print('[');
      while (head.next != null) {
          System.out.print(head.val + ", ");
          head = head.next;
      }
      System.out.println(head.val + "]");
  }
}
